frontend/jupyter/yarn.lock

```diff
@@ -6269,12 +6269,6 @@
   "resolved" "https://registry.npmjs.org/klona/-/klona-2.0.6.tgz"
   "version" "2.0.6"
 
-"kubeflow@file:../common/kubeflow-common-lib/dist/kubeflow":
```

Comment:
I don't know how much this actually would affect us, but it's probably best to NOT commit this change.